psalm-baseline.xml

```diff
@@ -519,6 +531,19 @@
       <code>hasStrategy</code>
       <code>removeStrategy</code>
     </UndefinedInterfaceMethod>
+    <DeprecatedMethod occurrences="2">
+      <code>$this->returnValue</code>
+    </DeprecatedMethod>
+  </file>
+  <file src="test/HydratorTestTrait.php">
+    <DeprecatedMethod occurrences="4">
+      <code>$this->returnValue</code>
+    </DeprecatedMethod>
+  </file>
+  <file src="test/NamingStrategy/CompositeNamingStrategyTest.php">
+    <DeprecatedMethod occurrences="4">
+      <code>$this->returnValue</code>
+    </DeprecatedMethod>
```

Comment:
These deprecations can be fixed in the test with `MockObject::willReturn($whatever)`